airgun/views/media.py

```diff
@@ -0,0 +1,64 @@
+from widgetastic.widget import Text, TextInput, View
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin, SatTab
+from airgun.widgets import FilteredDropdown, MultiSelect, SatTable
+
+
+class MediasView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Installation Media']")
+    new = Text("//a[contains(@href, '/media/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Actions': Text('.//a[@data-method="delete"]'),
```

Comment:
Why not Button? Are we gonna enforce using buttons btw or it still remains optional?

Reply:
That is probably under discussion. I would suggest to use Button for all unique buttons that can be addressed without xpath and in all places where you actually need to track events and etc.
Yeah, we can discuss that later